src/RefractAPI.cc

```diff
@@ -6,7 +6,6 @@
 //  Copyright (c) 2015 Apiary Inc. All rights reserved.
 //
 
-#include <iterator>
```

Comment:
In short? In C/C++ is prefered system down :)

In long:
```
// file a.h
std::string fn();
```

```
// file a.cc
#include <string>
#include "a.h"

std::string fn() { return "hello"; }
```

`a.cc` will be compiled ok

but
```
// file: b.cc
#include "a.h"

something...
```

compilation will fail with something like `unknown type 'std::string'`